packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -574,7 +574,7 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
             abi.encodePacked(
                 uint256(0), // mint
                 uint256(0), // value
-                uint64(SYSTEM_DEPOSIT_GAS_LIMIT), // gasLimit
+                uint64(10 * SYSTEM_DEPOSIT_GAS_LIMIT), // gasLimit
```

Comment:
I believe there is a bug here, the deposit transaction runs out of gas and this causes the test assertions around L1Block and WETH contracts to fail. This may not be the best fix, but it is sufficient to get the tests passing.

Reply:
If you could log the gas used of this tx that would be super helpful. It emits an event on L2 that you can search for

Reply:
```
    custom_gas_token_test.go:455: RECEIPT &{Type:126 PostState:[] Status:1 CumulativeGasUsed:146661 Bloom:[0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 32 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 4 0 0 0 0 0 0 0 0 0 0 0 10 0 0 0 16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 4 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 128 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0] Logs:[0x14007f92a50] TxHash:0x11eff4337d692df26fa8db89a0764cead90d1adab705afd5cf99debd434dc5e4 ContractAddress:0x0000000000000000000000000000000000000000
    GasUsed:96185 
EffectiveGasPrice:+0 BlobGasUsed:0 BlobGasPrice:<nil> DepositNonce:0x1400862c788 DepositReceiptVersion:0x1400862c790 BlockHash:0x53f3d61d69f57c53e1b0a67511fde32e970bcf68ec3cb3b989435c3ce0f71782 BlockNumber:+22 TransactionIndex:1 L1GasPrice:<nil> L1BlobBaseFee:<nil> L1GasUsed:<nil> L1Fee:<nil> FeeScalar:<nil> L1BaseFeeScalar:<nil> L1BlobBaseFeeScalar:<nil>}
```

Reply:
Should be fixed by https://github.com/ethereum-optimism/optimism/pull/10449